OBSDOCS-1284: Logging 5.9.6 Release Notes (#81473)
openshift/openshift-docs: modules/logging-release-notes-5-9-6.adoc

// module included in logging-5-9-release-notes.adoc
:_mod-docs-content-type: REFERENCE
[id="cluster-logging-release-notes-5-9-6_{context}"]
= Logging 5.9.6

> This release includes a link:https://access.redhat.com/errata/RHBA-2024:6095[OpenShift Logging Bug Fix Release 5.9.6].

Author: I am removing the article, as I used to do before, if that's okay.

[id="openshift-logging-5-9-6-bug-fixes_{context}"]
== Bug fixes

> * Before this update, the collector deployment ignored secret changes, causing receivers to reject logs. With this update, the system rolls out a new pod when there is a change in the secret value, ensuring the collector reloads the updated secrets. (link:https://issues.redhat.com/browse/LOG-5525[LOG-5525])

Reviewer: ```suggestion
* Before this update, the collector deployment ignored secret changes, causing receivers to reject logs. With this update, the system rolls out a new pod when there is a change in the secret value, ensuring that the collector reloads the updated secrets. (link:https://issues.redhat.com/browse/LOG-5525[LOG-5525])
```

* Before this update, the Vector could not correctly parse field values that included a single dollar sign (`$`). With this update, field values with a single dollar sign are automatically changed to two dollar signs (`$$`), ensuring proper parsing by the Vector. (link:https://issues.redhat.com/browse/LOG-5602[LOG-5602])

* Before this update, the drop filter could not handle non-string values (e.g., `.responseStatus.code: 403`). With this update, the drop filter now works properly with these values. (link:https://issues.redhat.com/browse/LOG-5815[LOG-5815])

* Before this update, the collector used the default settings to collect audit logs, without handling back the load from output receivers. With this update, the process for collecting audit logs has been improved to better manage file handles and log reading efficiency. (link:https://issues.redhat.com/browse/LOG-5866[LOG-5866])

> * Before this update, the `must-gather` tool failed on clusters with non-AMD64 architectures like Azure Resource Manager (ARM) or PowerPC. With this update, the tool now detects the cluster's architecture at runtime and uses architecture-independent paths and dependencies, allowing it to run smoothly on platforms like ARM and PowerPC. (link:https://issues.redhat.com/browse/LOG-5997[LOG-5997])

Reviewer: ```suggestion
* Before this update, the `must-gather` tool failed on clusters with non-AMD64 architectures such as Azure Resource Manager (ARM) or PowerPC. With this update, the tool now detects the cluster architecture at runtime and uses architecture-independent paths and dependencies. The detection allows `must-gather` to run smoothly on platforms like ARM and PowerPC. (link:https://issues.redhat.com/browse/LOG-5997[LOG-5997])
```

> * Before this update, the log level was set using a mix of structured and unstructured keywords that was unclear. With this update, the log level follows a clear, documented order, starting with structured keywords and then moving to unstructured ones, if needed. (link:https://issues.redhat.com/browse/LOG-6016[LOG-6016])

Reviewer: ```suggestion
* Before this update, the log level was set using a mix of structured and unstructured keywords that were unclear. With this update, the log level follows a clear, documented order, starting with structured keywords. (link:https://issues.redhat.com/browse/LOG-6016[LOG-6016])
```
Fairly obvious that unstructured would come next. Are there any other type of keywords? If so, just saying where it starts is a bit more evergreen.

Author: I agree, but I couldn't find any other keyword related to it. I'm using your suggestion.
